frontend/src/App.js

```diff
@@ -1,42 +1,30 @@
 import React, { Component } from 'react';
 import logo from './logo.png';
 import './App.css';
-
-const headers = new Headers({
-  Accept: 'application/json',
-  'Content-Type': 'application/json',
-});
+import Home from './Home.jsx';
+import Experiment from './Experiment.jsx';
+import Status from './Status.jsx';
+import { BrowserRouter as Router, Route, Link } from "react-router-dom";
```

Comment:
Here you have an *eslint* error:
![image](https://user-images.githubusercontent.com/23021242/51538116-6bec2100-1e1e-11e9-85ba-bcf4b63870a9.png)

Can we fix that too?